python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -19,8 +19,27 @@ import { streamingApiChain } from "./utils/oboeHelpers";
 import { datadogLogs } from "@datadog/browser-logs";
 import { LogEvent, LogEventData } from "./shared/types";
 
-export default function Editor() {
+const useStyles = createStyles(() => ({
+  editorBackground: {
+    background:
+      "radial-gradient(ellipse at top,#08122d,#030712),radial-gradient(ellipse at bottom,#030712,#030712)",
+    margin: "0 auto",
+    minHeight: "100vh",
+  },
+
+  logo: {
+    maxWidth: "80rem",
+    margin: "0 auto",
+    padding: "32px 0 0 32px",
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+}));
+
```

Comment:
This sets the logo/top bar to local editor theme for every mode right? Is this intended?

<img width="1918" alt="Screenshot 2024-01-17 at 12 14 22 PM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/e52f7c9c-e635-4e5d-8870-440f7edfcc6b">

Reply:
Ya, it's intended in order to show the image in local editor. It's only in our own local editor component, where mode will always be 'local', so shouldn't be a problem